### PR TITLE
chore(bridge-ui-v2): Modal and alerts to inform users about delay for L2->L1 bridging

### PR DESCRIPTION
**Pull request by KorbinianK**
### Dismissable popup (will only show once per device):

image
image

### ETH/ERC20 Bridge warning
image
image

# NFT Bridge warning
image
image

### Transaction warning
image
image

### CODE REVIEW

File: packages/bridge-ui-v2/src/components/WelcomeModal/WelcomeModal.svelte [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        {@html $t('bridge.welcome_modal.body')}
+      </p>
+
+      <div class="f-row w-full space-x-[25px]">
+        <ActionButton priority="primary" class="w-full" on:click={confirmModal}>
+          {$t('bridge.welcome_modal.confirm')}
+        </ActionButton>
+      </div>
+      <div class="w-full text-center">
+        <a href="https://www.taiko.xyz" target="_blank" class="link">{$t('bridge.welcome_modal.link')} </a>
```
Comment (KorbinianK) [inline packages/bridge-ui-v2/src/components/WelcomeModal/WelcomeModal.svelte:41]: Where should we link this to?

**Review by cyberhorsey — APPROVED** [review]

File: packages/bridge-ui-v2/src/i18n/en.json [hunk 2]
```diff
@@ -147,6 +150,12 @@
         "recipient": "Recipient",
         "review": "Review"
       }
+    },
+    "welcome_modal": {
+      "body": "Welcome to <span class='font-bold text-primary-brand'>Katla, our final testnet</span>.<br/>Strengthening security may cause a some delay in L1→L2 bridging, <span class='font-bold text-primary-brand'>around 24 hours</span> on the testnet. We're striving for shorter delays on the mainnet.",
```
Comment (cyberhorsey) [inline packages/bridge-ui-v2/src/i18n/en.json:155]: this says L1 to L2, should be L2 to L1

**Review by cyberhorsey — CHANGES_REQUESTED** [review]

File: packages/bridge-ui-v2/src/i18n/en.json (hunk 2 above)
Comment (KorbinianK) [inline packages/bridge-ui-v2/src/i18n/en.json:155]:
```suggestion
      "body": "Welcome to <span class='font-bold text-primary-brand'>Katla, our final testnet</span>.<br/>Strengthening security may cause a some delay in L2→L1 bridging, <span class='font-bold text-primary-brand'>around 24 hours</span> on the testnet. We're striving for shorter delays on the mainnet.",
```

File: packages/bridge-ui-v2/src/i18n/en.json [hunk 3]
```diff
@@ -33,6 +33,9 @@
       "nft_scan": "Scan for NFT",
       "nft_scan_again": "Scan again"
     },
+    "alerts": {
+      "slow_bridging": "Please be aware, bridging back to L1 will take around 24hs! Read our Katla announcement for more details."
```
Comment (KorbinianK) [inline packages/bridge-ui-v2/src/i18n/en.json:37]:
```suggestion
      "slow_bridging": "Please be aware, bridging back to L1 will take around 24hrs! Read our Katla announcement for more details."
```

File: packages/bridge-ui-v2/src/i18n/en.json (hunk 2 above)
Comment (KorbinianK) [reply packages/bridge-ui-v2/src/i18n/en.json]: fixed

**Review by cyberhorsey — APPROVED** [review]

**Review by davidtaikocha — APPROVED** [review]

**Pull request merged**